easybuild/easyconfigs/o/OptiX/OptiX-7.2.0.eb

```diff
@@ -0,0 +1,33 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+easyblock = 'Binary'
+
+name = 'OptiX'
+version = '7.2.0'
+
+homepage = 'https://developer.nvidia.com/optix'
+description = """OptiX is NVIDIA SDK for easy ray tracing performance.
+ It provides a simple framework for accessing the GPU’s massive ray tracing
+ power using state-of-the-art GPU algorithms."""
+
+toolchain = SYSTEM
+
+# Registration required. Download links:
+# https://developer.nvidia.com/designworks/optix/download
+# https://developer.nvidia.com/designworks/optix/downloads/legacy
+sources = ['NVIDIA-OptiX-SDK-%(version)s-linux64-x86_64.sh']
+checksums = ['6355a03b2819a3720276c0ca9358641635caf99b8e760b43d9a6e04e1ec78b85']
+
+install_cmd = "./" + sources[0] + " --skip-license --prefix=%(installdir)s"
+
+sanity_check_paths = {
+    'files': ["include/optix.h", "include/optix_device.h"],
+#    'files': ["include/optix.h", "include/optix_device.h", "lib64/liboptix.%s" % SHLIB_EXT,
+#              "lib64/liboptixu.%s" % SHLIB_EXT],
```

Comment:
does these not exist anymore? surely there is some library we can check for at least.

Reply:
The installer script of the previous release of OptiX, 6.5.0, created a lib64 directory, while the newest one, 7.2.0, doesn't. Searching inside of the installation directory structure doesn't show any libraries, shared or static.

Reply:
That is the reason why I commented it out in the first place. I tried to use it with VMD but, as expected, it doesn't work.

Reply:
Seems like optix 7 is is a complete redesign and is baked into the nvidia *drivers* (`/usr/lib64/libnvoptix.so.460.32.03`). Not sure what the "SDK" does anymore.

Reply:
https://forums.developer.nvidia.com/t/optix-6-5-cannot-recognize-device/157877/5  alright so, they confirmed that Optix is header only starting with optix 7. So. Just remove those commented lines and this PR should be good to go.

Reply:
@Micket Done.